game/libda/portaudio.cc

```diff
@@ -0,0 +1,120 @@
+#include "portaudio.hpp"
+
+#include "../platform.hh"
+
+namespace portaudio {
+
+	struct AudioBackend::AudioDevices {
+		/// Constructor gets the PA devices into a vector
+		AudioDevices(PaHostApiTypeId backend) {
+			auto backendIndex = Pa_HostApiTypeIdToHostApiIndex(backend);
```

Comment:
Please try not to use `auto` for these things, as it turns out it's not as intelligent as it should be sometimes.
Prefer portaudio's type aliases in this case.

Reply:
IMO not true. Left side is always the same type as on the right side. `auto` is the right decision even for external libs.